docs/subnets/add-subnet.md

```diff
@@ -16,8 +16,15 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Networks` card
 1. Click `View` in the `Networks` card, to see a list of existing `Networks`
-1. In the list of networks, find the network you want to manage and click `DETAILS`
+1. In the list of networks find the network where you want to add a subnet and click the `NAME` of the network
 1. On the upper-right corner of network details view, click `SUBNETS`
 1. On the upper-right corner of subnets list, click `ADD SUBNET`
-1. Follow the prompts then click `CONFIRM`
+1. Click `NEXT`
+1. Provide a name to the Subnet being added. Example: "Public Subnet A", "Private Subnet B"
+1. Select a `Type`
+1. Click `NEXT`
+1. Select a `ZONE` and click `NEXT`
```

Comment:
Some fields in the `Add subnet` form are cloud provider specific.
For example: on `AWS` a `Zone` is required to create a subnet, whereas on `GCP` a `Zone` is not required but a `Region` is required to create a subnet.
In cases like this perhaps makes more sense to use a generic text such as 
```
Follow the prompts then click ...
```
as used in the docs for [Add a Network](https://www.devopness.com/docs/networks/add-network/)?

Reply:
Ok, fixed.